PhysicsTools/PatAlgos/python/recoLayer0/bTagging_cff.py

```diff
@@ -38,6 +38,7 @@
   , 'pfDeepCMVAPositiveTagInfos'
     # TopTagInfos (unrelated to b tagging)
   , 'caTopTagInfos'
+  , 'pfDeepTagInfos'
```

Comment:
is this spelled correctly? shouldn't it be ```pfDeepFlavourTagInfos ``` ?

Reply:
that is right! added in https://github.com/cms-sw/cmssw/pull/19893/commits/c5b8b4b6485d358ec0fb2536e29b6f42550a2262